packages/react/src/FormattingToolbar/components/DefaultButtons/TextAlignButton.tsx

```diff
@@ -22,10 +23,30 @@ const icons: Record<TextAlignment, IconType> = {
   justify: RiAlignJustify,
 };
 
-export const TextAlignButton = (props: {
-  editor: BlockNoteEditor<DefaultBlockSchema>;
+export const TextAlignButton = <BSchema extends BlockSchema>(props: {
+  editor: BlockNoteEditor<BSchema>;
   textAlignment: TextAlignment;
 }) => {
+  const [show, setShow] = useState(true);
+
+  useEffect(() => {
```

Comment:
I think this is cleaner as `useMemo`, and get rid of `useState`. Because you never call `setShow` outside of the effect anyway